treetracker/development/treetracker-ios-testing-instructions.md

```diff
@@ -0,0 +1,20 @@
+# TreeTracker iOS Testing Instructions
+
+1. Request an email 'invitation' from iOS Team to the testing environment
+2. Download the TestFlight app on an iPhone
+3. Press the 'Redeem' button on the top right corner and enter the invitation code from the email
+4. Download the corresponding build on the phone
+5. Sign up and login to the Treetracker testing app
+6. Check ability to change users
+7. Use app to capture image and record (Note: The app has a block for GPS accuracy. The user should be able to get GPS reception, "add a tree." Take a photo and upload the image.)
+    * time per capture
+    * any bugs
+    * any unclear UX/UI
+8. Sync data
+9. Confirm data reaches the cloud by:
+    * checking that the images should appear on the map at test.treetracker.org
+    * logging into the Tree Tracker Admin URL, https://test-admin.treetracker.org/
+        * username: admin
+        * password: ZwovDoOxCo
```

Comment:
It's better not to put the credentials here. It should be available in the Slack channel.

Reply:
Okay, I'll change this part. By the way, this is on the documentation for android testing instructions as well.

Reply:
Just updated it.

Reply:
@CorrWu Thank you for making the changes. This PR will be on hold for a bit until Gitbook is synced properly and some other issues are resolved. I will merge it after that.

Reply:
@pranavkparti what else do we need before we can merge this?